crypto/evp/evp_utils.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2000-2019 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
fix copyright year

Reply:
it will get fixed by the scripts when a relese is done, ya know.

Reply:
Err.. Are you saying it looks at the creation date of the file.. and then changes the Copyright to just be Copyright 2019 instead of Copyright 2000-2019. And then it commits the updated file back to master?

Reply:
no, the release tools do the right thing; see https://github.com/openssl/tools/blob/master/release-tools/do-copyright-year#L18

Reply:
It looks like it just updates the second year field. If it does this its wrong. The copyright for this file should not contain 2000

Reply:
If this file is derived from a file that was copyright 2000 then the year is correct. @levitte will have to comment on the provenance/start-date of that file.

Reply:
I see no reason why the contents of this file aren't part of evp_lib

Reply:
That file is originally written by me, for this PR, so @slontis is entirely correct.

Regarding evp_lib.c, I'd rather not put these entirely internal utilities in there, it has already enough stuff piled in as it is.

Reply:
Copyright year fixed